app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Venue.java

```diff
@@ -16,16 +16,48 @@
     @ColumnInfo(name = "name")
     @SerializedName("name")
     String venueName;
+    @ColumnInfo(name = "short_name")
+    @SerializedName("short_name")
+    String venueShortName;
+    @ColumnInfo(name = "description")
+    @SerializedName("description")
+    String venueDescripion;
+    @ColumnInfo(name = "parent")
+    @SerializedName("parent")
+    String venueParentId;
+    @ColumnInfo(name = "parent_relation")
+    @SerializedName("parent_relation")
+    String venueParentRelation;
+    @ColumnInfo(name = "group_id")
+    @SerializedName("group_id")
+    Integer venueGroupId;
+    @ColumnInfo(name = "pixel_x")
+    @SerializedName("pixel_x")
+    Integer venuePixelX;
+    @ColumnInfo(name = "pixel_y")
+    @SerializedName("pixel_y")
+    Integer venuePixelY;
+    @ColumnInfo(name = "reusable")
+    @SerializedName("reusable")
+    Boolean venueReusable;
```

Comment:
Shouldn't there be a Venue object in place of all these?

Reply:
This is the venue model, right? Didn't really catch that.

Reply:
There is, although you'll have to edit it to include short names and other new fields

Reply:
Really confused now. Isn't that exactly what I've done?

Reply:
Misread, my bad